src/data.js

```diff
@@ -48,6 +48,31 @@ module.exports = [
       'Wordpress',
     ],
   },
+  {
+    name: '𝐁𝐞𝐧𝐧𝐞𝐭𝐭 𝐁𝐞𝐧𝐞𝐝𝐢𝐜𝐭',
+    description: '𝐒𝐨𝐟𝐭𝐰𝐚𝐫𝐞 𝐃𝐞𝐯𝐞𝐥𝐨𝐩𝐞𝐫 𝐚𝐧𝐝 𝐖𝐞𝐢𝐫𝐝𝐨',
+    url: 'baharajr.me/uses/',
```

Comment:
failed to ping `baharajr.me/uses/`

Reply:
Got that boss, I was actually working on it and something came up at work